Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
Is this the right place for `yarn check`? I was thinking it would be part of the `lint` target. I'd like `make lint` to fail for me locally were I to update a dependency in __package.json__ without regenerating __yarn.lock__. This leads naturally to the question of which command one runs to update __yarn.lock__. Let's make it clear: `make yarn.lock`.